packages/@tinacms/graphql/src/index.ts

```diff
@@ -48,7 +48,7 @@ export const buildSchema = async (
   const config = await fs
     .readFileSync(path.join(tempConfig, 'schema.json'))
     .toString()
-  await fs.rm(tempConfig, { recursive: true })
+  fs.rmSync(tempConfig, { recursive: true, force: true })
```

Comment:
Should this be awaited?
```suggestion
  await fs.rmSync(tempConfig, { recursive: true, force: true })
```

Reply:
It looks like it's just a synchronous function that returns void:

> Synchronously removes files and directories (modeled on the standard POSIX rm utility).